frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -76,7 +76,7 @@ export default class DocumentListToolbar extends Component {
 
     return (
       <Toolbar>
-        {peers && peers.length &&
+        {Boolean(peers && peers.length) &&
```

Comment:
Ah okay. I think `peers && (peers.length > 0)` would be more descriptive, but I guess that's just personal taste.